client/sites-dashboard/components/sites-site-item-thumbnail.tsx

```diff
@@ -43,16 +43,11 @@ export const SiteItemThumbnail = ( { site, ...props }: SiteItemThumbnailProps )
 };
 
 function getFirstGrapheme( input: string ) {
-	// TODO: once we're on Typescript 4.7 we should be able to add this comment:
-	//    /// <reference lib="es2022.intl" />
-	// to the top of the file to get access to the types for Intl.Segmenter
-	// which where added in microsoft/TypeScript#48800
-	// In the mean time we need to use the `any` type to fix type errors in CI.
-
 	try {
-		const segmenter = new ( Intl as any ).Segmenter();
-		const segments = segmenter.segment( input );
-		return ( Array.from( segments )[ 0 ] as any ).segment;
+		const segmenter = new Intl.Segmenter();
+		const [ firstSegmentData ] = segmenter.segment( input );
```

Comment:
TIL that destructuring assignment works on iterables! I wasn't sure initially (since access by index like `[0]` doesn't work), but it tests well 👍

Reply:
And more proof: https://bvaughn.github.io/babel-repl/#?babili=false&browsers=&builtIns=false&code_lz=MYewdgzgLgBBCmBzAtvMV4CcYF4ZngHcYBJdAGwDoBlJVdLACgEoBuAKHdElgG0AzAJaZotFGigBdXHDoSslBOPSMYAcjUw2QA&debug=false&evaluate=false&lineWrap=true&presets=es2015%2Cenv&prettier=false&showSidebar=true&targets=